tests/test_functors.py

```diff
@@ -581,14 +584,159 @@ def testDipPhotometry(self):
                                     atol=1e-13,
                                     rtol=0))
 
+    def testComputePositionAngle(self):
+        """Test computation of position angle from (RA1, Dec1) to (RA2, Dec2)
+        """
+
+        d = 0.0001  # [rad] offset amount, length of our vector
```

Comment:
I'd recommend 1) making this an optional argument to `testComputePositionAngle` and 2) use a longer variable name, at least in the function signature, that can be referred to as `d` inside this test later on.